### PR TITLE
Add HA-NFD PR

### PR DESCRIPTION
**Pull request by hle2**
Add PR to support hardware awareness based application scheduler.

### CODE REVIEW

File: content/en/proposals/ha-nfd.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Proposal-NNNN: Hardware Awareness scheduler based on NFD
```
Comment (dixudx) [inline content/en/proposals/ha-nfd.md:1]: We need to insert metadata at the beginning of this file. That's what the website theme requires.

```
---
title: "Hardware Awareness Scheduling Based on NFD"
description: "proposal for hardware awareness scheduling"
date: 2022-07-15
draft: false
weight: -2
---
```

**Review by dixudx — COMMENTED** [review]
Awesome proposal. Thanks for the contribution.

**Comment by lmxia** [comment]
lgtm.

**Review by dixudx — APPROVED** [review]
Great! Thanks for the proposal.

**Pull request merged**